### PR TITLE
make it easier to dbg stalls

### PR DESCRIPTION
**Pull request by ordian**
cc #3242

this is rolled out on two rococo nodes, but no stalls so far

### CODE REVIEW

File: node/overseer/src/lib.rs [hunk 1]
```diff
@@ -989,7 +1001,7 @@ impl<M> OverseenSubsystem<M> {
 	/// Send a message to the wrapped subsystem.
 	///
 	/// If the inner `instance` is `None`, nothing is happening.
-	async fn send_message(&mut self, msg: M) -> SubsystemResult<()> {
+	async fn send_message(&mut self, msg: M, origin: &'static str) -> SubsystemResult<()> {
```
Comment (drahnr) [inline node/overseer/src/lib.rs:1004]: Should we add a secondary ` fn send_message_anon(&mut self, msg: M)` to avoid the clutter through the source?

Comment (ordian) [reply node/overseer/src/lib.rs:1004]: the anon would only be used in tests in one file, I'm not sure it's worth it

File: node/overseer/src/lib.rs [hunk 2]
```diff
@@ -102,9 +102,9 @@ use polkadot_procmacro_overseer_subsystems_gen::AllSubsystemsGen;
 mod tests;
 
 // A capacity of bounded channels inside the overseer.
-const CHANNEL_CAPACITY: usize = 1024;
+const CHANNEL_CAPACITY: usize = 128;
 // The capacity of signal channels to subsystems.
-const SIGNAL_CHANNEL_CAPACITY: usize = 64;
+const SIGNAL_CHANNEL_CAPACITY: usize = 16;
```
Comment (ordian) [inline node/overseer/src/lib.rs:107]:
```suggestion
const SIGNAL_CHANNEL_CAPACITY: usize = 64;
```

File: node/overseer/src/lib.rs [hunk 3]
```diff
@@ -102,9 +102,9 @@ use polkadot_procmacro_overseer_subsystems_gen::AllSubsystemsGen;
 mod tests;
 
 // A capacity of bounded channels inside the overseer.
-const CHANNEL_CAPACITY: usize = 1024;
+const CHANNEL_CAPACITY: usize = 128;
```
Comment (ordian) [inline node/overseer/src/lib.rs:105]:
```suggestion
const CHANNEL_CAPACITY: usize = 1024;
```

File: node/overseer/src/lib.rs (hunk 1 above)
Comment (drahnr) [reply node/overseer/src/lib.rs:1004]: Your call, just thought it's a small addition for some easy IDE aided lookups, your call :)

**Review by drahnr — APPROVED** [review]
LGTM!

**Comment by ordian** [comment]
@s3krit would be nice to get this in 0.9.8

**Pull request merged**